tools/make/kube.mk

```diff
@@ -126,16 +126,20 @@ run-e2e: install-e2e-telemetry
 	kubectl apply -f test/config/gatewayclass.yaml
 ifeq ($(E2E_RUN_TEST),)
 	go test -v -tags e2e ./test/e2e --gateway-class=envoy-gateway --debug=true --cleanup-base-resources=false
-	go test -v -tags e2e ./test/e2e/upgrade --gateway-class=upgrade --debug=true --cleanup-base-resources=$(E2E_CLEANUP)
+	go test -v -tags e2e ./test/e2e/upgrade --gateway-class=upgrade --debug=true --cleanup-base-resources=false
```

Comment:
we should run upgrade test in the end because it recreates (deletes and adds) the installations resources

Reply:
done